src/publish/cancelled.js

```diff
@@ -0,0 +1,56 @@
+exports.default = async function cancelled({
+  context,
+  github,
+  inputs,
+  Sentry,
+}) {
+  const { repo, version } = inputs;
+
+  const repoInfo = context.repo;
```

Comment:
```suggestion
  const { repo: repoInfo } = context;
```

Reply:
~~If we get `repoInfo` from context, which should always be `publish`, what are we doing with `repo` above?~~

Okay, I now see what you use these for but the naming is very confusing. I'd name this one `publishRepo` or something.